packages/fxa-auth-server/lib/routes/validators.js

```diff
@@ -302,6 +302,7 @@ module.exports.activeSubscriptionValidator = isA.object({
 
 // This is subhub's perspective on an active subscription
 module.exports.subscriptionsSubscriptionValidator = isA.object({
+  created: isA.number().required(),
```

Comment:
Would it be worthwhile to tighten this to some kind of timestamp value? Or at least a number of digits (so it's clear this is a millisecond vs second timestamp)?

Reply:
can do

Reply:
should I update all of the other timestamp instances in that response validator?

Reply:
So, for this response, all of the timestamps are seconds since the Unix epoch. What do you think would be a good validation requirement for this, since technically the size will grow?

Reply:
I think it'd be fine to just improve this one line, and just aim to improve the other validators the next time they are edited.

Joi has some [timestamp validation built-ins](https://github.com/hapijs/joi/blob/v9.0.0-3/API.md#datetimestamptype) that could help. I'm now a little confused about whether this is a JS or unix timestamp, but you could do `Joi.date().timestamp()` for millisecond timestamps, or `Joi.date().timestamp('unix')` for second timestamps.